tests/test_session_cookies.py

```diff
@@ -0,0 +1,16 @@
+import requests
+import requests_mock
+
+from . import base
+
+
+class SessionCookiesTest(base.TestCase):
+    def test_cookie(self):
```

Comment:
Another test is required that contains two cookies, as the behavior only seems to work for one cookie. See https://github.com/jamielennox/requests-mock/issues/17#issuecomment-1372546470

Reply:
[This answer](https://stackoverflow.com/a/2880070/70170) adds some insight, indicating to me that semicolon separated values aren't appropriate for `Set-Cookie`. I tried comma-separated values, but that didn't work either... and I've not found a way to indicate more than one Set-Cookie header (because `response.headers` is a dict).

Reply:
Fixed in 4ce92fadc.